### PR TITLE
Test new github features

### PR DESCRIPTION
**Pull request by robertpainsi**
Test

### CODE REVIEW

File: catroid/src/main/java/org/catrobat/catroid/ui/fragment/FormulaEditorCategoryListFragment.java [hunk 1]
```diff
@@ -162,7 +162,7 @@
 			R.string.formula_editor_sensor_drone_emergency_state, R.string.formula_editor_sensor_drone_flying,
 			R.string.formula_editor_sensor_drone_initialized, R.string.formula_editor_sensor_drone_usb_active,
 			R.string.formula_editor_sensor_drone_usb_remaining_time, R.string.formula_editor_sensor_drone_camera_ready,
-			R.string.formula_editor_sensor_drone_record_ready, R.string.formula_editor_sensor_drone_recording,
+			R.string.formula_editor_sensor_drone_record_sadadready, R.string.formula_editor_sensor_drone_recording,
```
Comment (robertpainsi) [inline catroid/src/main/java/org/catrobat/catroid/ui/fragment/FormulaEditorCategoryListFragment.java:165]: Test

File: catroid/src/main/java/org/catrobat/catroid/ui/fragment/FormulaEditorCategoryListFragment.java [hunk 2]
```diff
@@ -251,7 +251,7 @@ private int getNXTPort(int port) {
 		switch (port) {
 			case 0:
 				return R.string.formula_editor_sensor_lego_nxt_1;
-			case 1:
+			case 1:das
```
Comment (robertpainsi) [inline catroid/src/main/java/org/catrobat/catroid/ui/fragment/FormulaEditorCategoryListFragment.java:254]: Das is das das das

File: catroid/src/main/java/org/catrobat/catroid/ui/fragment/FormulaEditorCategoryListFragment.java [hunk 3]
```diff
@@ -442,10 +442,16 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 		return inflater.inflate(R.layout.fragment_formula_editor_list, container, false);
 	}
 
-	public void showFragment(Context context) {
+	public void showFragment(Context context) {asdad
 		Activity activity = (Activity) context;
-		FragmentManager fragmentManager = activity.getFragmentManager();
-		FragmentTransaction fragTransaction = fragmentManager.beginTransaction();
+		FragmentManager fragmentManager = activityasd.getFragmentManager();
```
Comment (robertpainsi) [inline catroid/src/main/java/org/catrobat/catroid/ui/fragment/FormulaEditorCategoryListFragment.java:447]: And das

File: catroid/src/main/java/org/catrobat/catroid/ui/fragment/FormulaEditorCategoryListFragment.java [hunk 4]
```diff
@@ -442,10 +442,16 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
...
+		FragmentTransaction fragTransactiod
+		
+		
+		
+		
+		
+		n = fragmentManager.beginTransaction();
```
Comment (robertpainsi) [inline catroid/src/main/java/org/catrobat/catroid/ui/fragment/FormulaEditorCategoryListFragment.java:454]: this is good

**Review by robertpainsi — COMMENTED** [review]
Done

**Review by robertwhitebit — CHANGES_REQUESTED** [review]
Please see @robertpainsi changes :smirk_cat:

**Review by robertwhitebit — CHANGES_REQUESTED** [review]
asdsd

**Review by robertpainsi — COMMENTED** [review]
asd

**Review by robertwhitebit — APPROVED** [review]
Looks good

**Pull request closed without merging**